.travis.yml

```diff
@@ -11,6 +11,7 @@ cache:
 before_install:
   - sudo apt-get install -y libgconf-2-4
 jobs:
+  fast_finish: true
```

Comment:
`matrix` is an alias for `jobs`. [`fast_finish` is valid here](https://docs.travis-ci.com/user/customizing-the-build/#fast-finishing).